tests/phpunit/elementor/modules/atomic-widgets/template-renderer/__snapshots__/Test_Template_Renderer__test_render__basic_template__1.txt

```diff
@@ -0,0 +1,6 @@
+			<div class="class1 class2">
```

Comment:
why do we have this gap here?

Reply:
It is done by elementor widget render, not something specific for twig.